mesheryctl/internal/cli/root/experimental/experimental.go

```diff
@@ -44,6 +45,6 @@ var ExpCmd = &cobra.Command{
 }
 
 func init() {
-	availableSubcommands = append(availableSubcommands, system.ModelCmd, system.ComponentsCmd)
+	availableSubcommands = append(availableSubcommands, system.ModelCmd, system.ComponentsCmd, connections.ConnectionsCmd)
```

Comment:
Please double check if `system.ComponentsCmd` is there or not. I believe I have migrated it completely to `components.ComponentsCmd`

Reply:
..